config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -13,22 +13,31 @@ periodics:
     preset-e2e-gci-gce-scalability: "true"
   spec:
     containers:
-    - args:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20181205-915278e90-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
       - --timeout=140
-      - --bare
```

Comment:
Why do we remove "bare" ?

Reply:
IIRC `bare` is to ensure that is no repository checkout. There is `bare` xor `repo` verification.
Clusterloader requires perf-tests repo.

Reply:
OK got it. Thanks.